packages/design-system/fieldset/index.tsx

```diff
@@ -48,60 +55,66 @@ export function Fieldset(props: FieldsetProps) {
   const inputId = useId();
   const helperTextId = useId();
   const errorTextId = useId();
-
+  const switchTw = switchOnly
+    ? "flex-1 flex-row items-center justify-between"
+    : "";
   return (
-    <View tw={`rounded-4 p-4 ${style} ${twProp}`}>
+    <View tw={`rounded-4 p-4 ${style} ${twProp} ${switchTw}`}>
       <View tw="flex-row">
         <Label htmlFor={inputId} tw="font-bold text-gray-900 dark:text-white">
           {label}
         </Label>
         {required ? <Text tw="ml-1 text-red-500">*</Text> : null}
```

Comment:
@intergalacticspacehighway just merged [your branch](https://github.com/showtime-xyz/showtime-frontend/pull/1447), I think it is correct, but I'm not sure if it will affect your feature, could help me check it? thanks! 🙏